zkevm-circuits/src/util/word.rs

```diff
@@ -54,14 +54,6 @@ impl<T: Default, const N: usize> Default for WordLimbs<T, N> {
     }
 }
 
-impl<F: Field> From<Word32Cell<F>> for WordLegacy<F> {
-    fn from(_: Word32Cell<F>) -> Self {
-        unreachable!(
-            "This function is meat to fix the build. Remove once we fixed the word lo-hi refactor"
-        )
-    }
-}
-
 impl<F: Field> Expr<F> for WordCell<F> {
     fn expr(&self) -> Expression<F> {
         unreachable!(
```

Comment:
Quick comment.
We can remove `impl<F: Field> Expr<F> for WordCell<F>` and `impl<F: Field> Expr<F> for Word32Cell<F>` too.

Reply:
Done, tks for reminding!